From 0e7a73b04ae95bad9fe01c3af3832c25f4123873 Mon Sep 17 00:00:00 2001
From: root <jeffbai@aosc.io>
Date: Tue, 18 Feb 2025 15:20:56 +0800
Subject: [PATCH 360/360] x86: configs: enable BT_INTEL_PCIE

Also refresh defconfig against the current tree.

Signed-off-by: root <jeffbai@aosc.io>
(cherry picked from commit 775a1884d2e770f2b5e8171ea2fb2472a8ea2d54)
---
 arch/x86/configs/deepin_x86_desktop_defconfig | 24 ++-----------------
 1 file changed, 2 insertions(+), 22 deletions(-)

diff --git a/arch/x86/configs/deepin_x86_desktop_defconfig b/arch/x86/configs/deepin_x86_desktop_defconfig
index 8696f3c9c4a5..f4499d46cf4c 100644
--- a/arch/x86/configs/deepin_x86_desktop_defconfig
+++ b/arch/x86/configs/deepin_x86_desktop_defconfig
@@ -57,9 +57,7 @@ CONFIG_KEXEC_FILE=y
 CONFIG_KEXEC_SIG=y
 CONFIG_KEXEC_BZIMAGE_VERIFY_SIG=y
 CONFIG_KEXEC_JUMP=y
-CONFIG_CRASH_DUMP=y
 CONFIG_SMP=y
-# CONFIG_GOLDFISH is not set
 CONFIG_X86_CPU_RESCTRL=y
 CONFIG_X86_NUMACHIP=y
 CONFIG_X86_VSMP=y
@@ -671,6 +669,7 @@ CONFIG_BT_MTKSDIO=m
 CONFIG_BT_MTKUART=m
 CONFIG_BT_VIRTIO=m
 CONFIG_BT_NXPUART=m
+CONFIG_BT_INTEL_PCIE=m
 CONFIG_AF_RXRPC_IPV6=y
 CONFIG_RXKAD=y
 CONFIG_RXPERF=m
@@ -2403,7 +2402,6 @@ CONFIG_SENSORS_NTC_THERMISTOR=m
 CONFIG_SENSORS_NCT6683=m
 CONFIG_SENSORS_NCT6775=m
 CONFIG_SENSORS_NCT6775_I2C=m
-CONFIG_SENSORS_NCT7511=m
 CONFIG_SENSORS_NCT7802=m
 CONFIG_SENSORS_NCT7904=m
 CONFIG_SENSORS_NPCM7XX=m
@@ -3493,10 +3491,6 @@ CONFIG_SND_SOC_IMG_SPDIF_OUT=m
 CONFIG_SND_SOC_IMG_PISTACHIO_INTERNAL_DAC=m
 CONFIG_SND_SOC_INTEL_CATPT=m
 CONFIG_SND_SST_ATOM_HIFI2_PLATFORM_PCI=m
-CONFIG_SND_SOC_INTEL_SKYLAKE=m
-CONFIG_SND_SOC_INTEL_CML_H=m
-CONFIG_SND_SOC_INTEL_CML_LP=m
-CONFIG_SND_SOC_INTEL_SKYLAKE_HDAUDIO_CODEC=y
 CONFIG_SND_SOC_INTEL_AVS=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_DA7219=m
 CONFIG_SND_SOC_INTEL_AVS_MACH_DMIC=m
@@ -3525,11 +3519,6 @@ CONFIG_SND_SOC_INTEL_BYT_CHT_CX2072X_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_DA7213_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_ES8316_MACH=m
 CONFIG_SND_SOC_INTEL_BYT_CHT_NOCODEC_MACH=m
-CONFIG_SND_SOC_INTEL_SKL_RT286_MACH=m
-CONFIG_SND_SOC_INTEL_SKL_NAU88L25_SSM4567_MACH=m
-CONFIG_SND_SOC_INTEL_SKL_NAU88L25_MAX98357A_MACH=m
-CONFIG_SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH=m
-CONFIG_SND_SOC_INTEL_BXT_RT298_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_WM8804_MACH=m
 CONFIG_SND_SOC_INTEL_GLK_DA7219_MAX98357A_MACH=m
 CONFIG_SND_SOC_INTEL_GLK_RT5682_MAX98357A_MACH=m
@@ -3540,7 +3529,6 @@ CONFIG_SND_SOC_INTEL_SOF_ES8336_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_NAU8825_MACH=m
 CONFIG_SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH=m
-CONFIG_SND_SOC_INTEL_SOF_DA7219_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_SSP_AMP_MACH=m
 CONFIG_SND_SOC_INTEL_EHL_RT5660_MACH=m
 CONFIG_SND_SOC_INTEL_SOUNDWIRE_SOF_MACH=m
@@ -3592,7 +3580,6 @@ CONFIG_SND_SOC_CS35L36=m
 CONFIG_SND_SOC_CS35L45_SPI=m
 CONFIG_SND_SOC_CS35L45_I2C=m
 CONFIG_SND_SOC_CS35L56_I2C=m
-CONFIG_SND_SOC_CS35L56_SPI=m
 CONFIG_SND_SOC_CS42L51_I2C=m
 CONFIG_SND_SOC_CS42L52=m
 CONFIG_SND_SOC_CS42L56=m
@@ -3617,7 +3604,6 @@ CONFIG_SND_SOC_ES8388=m
 CONFIG_SND_SOC_GTM601=m
 CONFIG_SND_SOC_ICS43432=m
 CONFIG_SND_SOC_IDT821034=m
-CONFIG_SND_SOC_INNO_RK3036=m
 CONFIG_SND_SOC_MAX98088=m
 CONFIG_SND_SOC_MAX98504=m
 CONFIG_SND_SOC_MAX9867=m
@@ -3639,12 +3625,10 @@ CONFIG_SND_SOC_PCM3168A_SPI=m
 CONFIG_SND_SOC_PCM5102A=m
 CONFIG_SND_SOC_PCM512x_SPI=m
 CONFIG_SND_SOC_PEB2466=m
-CONFIG_SND_SOC_RK3328=m
 CONFIG_SND_SOC_RT1017_SDCA_SDW=m
 CONFIG_SND_SOC_RT5616=m
 CONFIG_SND_SOC_RT5631=m
 CONFIG_SND_SOC_RT5659=m
-CONFIG_SND_SOC_RT722_SDCA_SDW=m
 CONFIG_SND_SOC_RT9120=m
 CONFIG_SND_SOC_SGTL5000=m
 CONFIG_SND_SOC_SIMPLE_AMPLIFIER=m
@@ -4598,6 +4582,7 @@ CONFIG_VME_BUS=y
 CONFIG_VME_TSI148=m
 CONFIG_VME_FAKE=m
 CONFIG_VME_USER=m
+# CONFIG_GOLDFISH is not set
 CONFIG_CHROME_PLATFORMS=y
 CONFIG_CHROMEOS_LAPTOP=m
 CONFIG_CHROMEOS_PSTORE=m
@@ -4679,8 +4664,6 @@ CONFIG_INTEL_VBTN=m
 CONFIG_INTEL_INT0002_VGPIO=m
 CONFIG_INTEL_OAKTRAIL=m
 CONFIG_INTEL_BXTWC_PMIC_TMU=m
-CONFIG_INTEL_BYTCRC_PWRSRC=m
-CONFIG_INTEL_CHTDC_TI_PWRBTN=m
 CONFIG_INTEL_CHTWC_INT33FE=m
 CONFIG_INTEL_ISHTP_ECLITE=m
 CONFIG_INTEL_MRFLD_PWRBTN=m
@@ -4841,7 +4824,6 @@ CONFIG_AD7923=m
 CONFIG_AD7949=m
 CONFIG_AD799X=m
 CONFIG_AD9467=m
-CONFIG_ADI_AXI_ADC=m
 CONFIG_AXP20X_ADC=m
 CONFIG_AXP288_ADC=m
 CONFIG_CC10001_ADC=m
@@ -5605,8 +5587,6 @@ CONFIG_BOOT_PRINTK_DELAY=y
 CONFIG_DYNAMIC_DEBUG=y
 # CONFIG_DEBUG_MISC is not set
 CONFIG_DEBUG_INFO_DWARF5=y
-CONFIG_DEBUG_INFO_BTF=y
-CONFIG_MODULE_ALLOW_BTF_MISMATCH=y
 CONFIG_GDB_SCRIPTS=y
 CONFIG_FRAME_WARN=4096
 CONFIG_STRIP_ASM_SYMS=y
-- 
2.20.1

